app/src/main/res/layout-sw600dp/help_activity.xml

```diff
@@ -10,9 +10,23 @@
     android:layout_height="match_parent"
     android:orientation="vertical">
 
-    <include
-      android:id="@+id/help_activity_toolbar"
-      layout="@layout/toolbar" />
```

Comment:
Can toolbar.xml be removed now?

Reply:
Yes, the toolbar can be removed and has been removed. PTAL